server/lib/cordova.js

```diff
@@ -0,0 +1,215 @@
+/* global Push, SystemLogger */
+
+Meteor.methods({
+	log() {
```

Comment:
What's the purpose of this method? Just because it could be used to spam the logs making it harder to find errors if someone found out about this and wanted to be evil.